### PR TITLE
test_sse1_full

### PR DESCRIPTION
**Pull request by juj**
Adds a full test of SSE1 API, currently disabled due to same reasons as test_sse1. This is something I had started writing earlier, but never had time to clean up for submitting until now. CC @kripken and @sunfishcode . I'm working on adding a similar test to cover SSE2 as well.

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -5501,6 +5500,19 @@ def test_sse1(self):
...
+  # Tests the full SSE1 API.
+  def test_sse1_full(self):
+    return self.skip('TODO: This test fails due to bugs #2840, #3044, #3045, #3046 and #3048 (also see #3043 and #3049)')
+    if SPIDERMONKEY_ENGINE not in JS_ENGINES: return self.skip('test_sse1_full requires SpiderMonkey to run.')
+    Popen([CLANG, path_from_root('tests', 'test_sse1_full.c'), '-o', 'test_sse1_full'] + get_clang_native_args(), stdout=PIPE, stderr=PIPE).communicate()
+    native_result, err = Popen('./test_sse1_full', stdout=PIPE, stderr=PIPE).communicate()
+
+    Settings.PRECISE_F32 = 1 # SIMD currently requires Math.fround
+    orig_args = self.emcc_args
+    for mode in [[], ['-s', 'SIMD=1']]:
```
Comment (kripken) [inline tests/test_core.py]: why test the SIMD flag? it's just for autovectorization i think. should not matter here.

**Comment by kripken** [comment]
I see this is still in skipped state, that's intentional?

File: tests/test_core.py (hunk 1 above)
Comment (juj) [inline tests/test_core.py]: I copied that from the other sse test, just for additional coverage, and indeed to test that nothing is thrown off here in either compilation mode.

**Comment by juj** [comment]
Yes, it's intentional to be skipped like `test_sse1` also currently is, since it does not actually work yet due to the different simd issues we have.

**Comment by kripken** [comment]
Ok, cool. Looks like @sunfishcode has no concerns either.

**Pull request merged**